iOS/DuckDuckGo/SettingsViewModel.swift

```diff
@@ -515,7 +517,8 @@ final class SettingsViewModel: ObservableObject {
          featureDiscovery: FeatureDiscovery = DefaultFeatureDiscovery(),
          subscriptionFreeTrialsHelper: SubscriptionFreeTrialsHelping = SubscriptionFreeTrialsHelper(),
          urlOpener: URLOpener = UIApplication.shared,
-         keyValueStore: ThrowingKeyValueStoring
+         keyValueStore: ThrowingKeyValueStoring,
+         dataBrokerProtectionIOSManager: DataBrokerProtectionIOSManager = .shared!
```

Comment:
I forgot to mention one important detail when we chatted earlier: this is disabled in production, so the force unwrap here will cause a crash in that case. Check out `isDBPStaticallyEnabled` if you want to see where this is defined.

For now we should make this property optional instead, and unwrap it when attempting to push the PIR flow.